fontforgeexe/cvpalettes.c

```diff
@@ -2058,7 +2111,12 @@ return;
     if ( cv->showback[0]&1 ) gcd[4].gd.flags |= gg_cb_on;
     if ( cv->showfore ) gcd[5].gd.flags |= gg_cb_on;
 
-    GGadgetsCreate(cvlayers2,gcd);
+    GGadgetsCreate(cvlayers2, gcd);
+    // Calculate the header height. The '-1' is magic! Needed to obtain symmetry
+    layer2.header_height = GGadgetGetY(GWidgetGetControl(cvlayers2, CID_VGrid)) - 1;
```

Comment:
I've moved the header height calculation here, since there's no need to recalculate it with every call to `CVLayers2Reflow`.